element/src/commonMain/kotlin/RootElement.kt

```diff
@@ -22,31 +31,54 @@ public class RootElement : Element() {
         children.forEach { it.draw(kanvas) }
     }
 
-    /** Returns true if an element was found and clicked on. Returns false if no element matched. */
-    public fun onClick(isPointInPath: IsPointInPath, x: Float, y: Float): Boolean {
-        // Union types would be pretty nice here. Value is of type T where T: Element and T: Interactable<T>
-        val interactable = visibilityOrderedDescendants()
-            .filterIsInstance<Interactable<*>>()
-            .filter { it.onClick != null }
-            .firstOrNull { interactable ->
-                val transform = (interactable as Element).totalTransform()
-                isPointInPath.isPointInPath(transform, interactable.getInteractionPath(), x, y)
-            }
-        val fallback = onClickFallback
-        return when {
-            interactable != null -> {
-                @Suppress("UNCHECKED_CAST") // Interactable<T> always accepts itself as the type argument.
-                val onClick = interactable.onClick as (Element) -> Unit
-                onClick(interactable as Element)
-                true
-            }
-            fallback != null -> {
-                fallback()
-                true
+    /**
+     * Entry point for dispatching hover events. Usually you won't need to call this, and it will be
+     * handled by your platform-specific ElementView.
+     */
+    public fun onHover(isPointInPath: IsPointInPath, x: Float, y: Float) {
```

Comment:
Would `onHoverStart` and `onHoverEnd` (or `onHoverFinish`) be more explicit?

Reply:
It's both start andmoved-while-hovering, not sure what a better name for it would be. I do like `onHoverEnd` though!